src/librbd/api/Group.cc

```diff
@@ -53,26 +53,16 @@ snap_t get_group_snap_id(I* ictx,
   return CEPH_NOSNAP;
 }
 
-int group_snap_list(librados::IoCtx& group_ioctx, const char *group_name,
+int group_snap_list(librados::IoCtx& group_ioctx, const std::string& group_id,
```

Comment:
This change seems unrelated to this PR. Why is it required?

Reply:
This is to avoid calling `dir_get_id` CLS method twice -- currently all but one caller of `group_snap_list()` do it.

Reply:
Correct